### ISSUE TITLE
Missing icons in High Contrast theme

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#10140]
- VSCode Version: 1.4.0
- OS Version: Windows

Steps to Reproduce:

1. Choose High Contrast theme from File -> Preferences -> Color Theme
2. Open search and observe the "Files to include/exclude" field. The icons are not visible there.

High Contrast:

![image](https://cloud.githubusercontent.com/assets/16890566/17417324/8ec11e4e-5a47-11e6-9b42-4871009ffe6f.png)

Dark theme:

![image](https://cloud.githubusercontent.com/assets/16890566/17417332/a1cb99f6-5a47-11e6-9fdd-45ee3fe9e24c.png)
